src/css/profile/wearable/changeable/theme-circle/listview.less

```diff
@@ -188,9 +188,34 @@
 				&.ui-li-grid-full {
 					justify-content: space-between;
 
-					div {
-						text-align: inherit;
+					// List divider "|" between text area and control button.
+					div > div {
+						text-align: center;
+
+						&.ui-marquee {
+							display: flex;
+							flex-direction: row;
+							max-width: 222 * @px_base;
+							max-height: 42 * @px_base;
+
+							&::after {
+								opacity: 1;
+								content: "";
+								position: relative;
+								width: 2 * @px_base;
+								height: 49 * @px_base;
+								top: 0;
+								right: 0;
+								margin-left: 14 * @px_base;
+								margin-right: 19 * @px_base;
+								background: rgba(255, 255, 255, 0.20);
```

Comment:
1. opacity is 19% not 20% in guide
2. could you define this as color variable e.g. in src/css/profile/wearable/changeable/theme-changeable/theme.color.less

Reply:
Done. Thanks.